jenkins/databricks/clusterutils.py

```diff
@@ -46,13 +46,18 @@ def generate_create_templ(sshKey, cluster_name, runtime, idle_timeout,
         templ['driver_node_type_id'] = driver_node_type
         templ['ssh_public_keys'] = [ sshKey ]
         templ['num_workers'] = num_workers
-        templ['init_scripts'] = [
-            {
-                "dbfs": {
-                     "destination": "dbfs:/databricks/init_scripts/init_cudf_udf.sh"
-                }
-            }
-        ]
+        if (init_script != ''):
```

Comment:
We'll add -f dbfs:/databricks/init_scripts/init_cudf_udf.sh,dbfs:/databricks/init_scripts/init_cuda11_runtime.sh in Jenkinsfile.databricks301nightly to init cudf_udf and cuda11